.github/fabricbot.json

```
@@ -3687,7 +3687,7 @@
             {
               "name": "bodyContains",
               "parameters": {
-                "bodyPattern": "(aot)/i",
+                "bodyPattern": "([Aa][Oo][Tt])+",
```

What kind of regex engine is being used? .NET? If it is, could this be written:

```suggestion
                "bodyPattern": "(?i)(?:aot)",
```

---

We clarified offline that they don't support options. 😢

---

How else do you do case insensitive? is there a parameter for it? Seems like a major limitation.

---

It is a major limitation indeed. Which they now know about !